src/constant/global.ts

```diff
@@ -274,6 +274,17 @@ const PORTFOLIO_TEMPLATES: PortfolioTemplateProps[] = [
   },
 ];
 
+//  For YouFocus Explore Page
+const Skills = [
+  'React.js',
+  'Node.js',
+  'AI',
+  'JavaScript',
+  'CSS',
+  'Three.js',
+  'Python',
+];
+
```

Comment:
![image](https://github.com/user-attachments/assets/3f88f63b-ca58-4100-b5a9-cf5682562c76)
Update this as per the task